tools/renesas-rcar-s4/make-rootfs.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+SCRIPT_DIR=$(cd `dirname $0` && pwd)
```

Comment:
Recommend to add `set -euo pipefail` to halt the script on the first error

Reply:
added